Plugins/Json/src/GridJsonConverter.cpp

```diff
@@ -24,3 +25,13 @@ nlohmann::json Acts::AxisJsonConverter::toJson(const IAxis& ia) {
   }
   return jAxis;
 }
+
+nlohmann::json Acts::AxisJsonConverter::toJsonDetray(const IAxis& ia) {
+  nlohmann::json jAxis;
+  jAxis["bounds"] =
+      ia.getBoundaryType() == Acts::detail::AxisBoundaryType::Bound ? 1 : 2;
+  jAxis["binning"] = ia.isEquidistant() ? 0 : 1;
+  jAxis["bins"] = ia.getNBins();
+  jAxis["edges"] = ia.getBinEdges();
```

Comment:
I am assuming these are {min, max} for regular binning?

Reply:
No they are all the edits, do you expect min/max ? in this case I will catch it here.

Reply:
For regular binning it is just min/max. For irregular binning it expects the lower bin edges (this will be the case e.g. for the volume grid)

Reply:
Ok, I will then catch the type and and write min/max in this case.

Reply:
Done.